### PR TITLE
Fix typo in buildkite-agent post-install notes

### PR DESCRIPTION
**Pull request by ddbeck**
Fixes https://github.com/buildkite/docs/issues/1496

### CODE REVIEW

File: buildkite-agent.rb [hunk 1]
```diff
@@ -105,7 +105,7 @@ def caveats
 
       To prevent your machine from sleeping or logging out, use `caffienate`. For
```
Comment (plaindocs) [inline buildkite-agent.rb:106]:
```suggestion
      To prevent your machine from sleeping or logging out, use `caffeinate`. For
```

**Comment by mckern** [comment]
Was about to open a PR for this 😅

**Comment by karensawrey** [comment]
Are we shipping it? ;)

**Comment by ddbeck** [comment]
I don't have merge privileges on this repository and I'm not sure who to ask for a merge. 😕

**Comment by plaindocs** [comment]
@JuanitoFatas can you do this? Or you do you know who can?

**Comment by karensawrey** [comment]
@JuanitoFatas *a gentle ping*

**Comment by karensawrey** [comment]
Asked the folks at Engineering. This should finally be merged :)

**Comment by JuanitoFatas** [comment]
Wow thanks so much everyone and sorry for the delay 👍🏼 💚

**Pull request merged**